presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -1218,4 +1216,56 @@ private static Session noPartialAggregationPushdown(Session session)
                 .setCatalogSessionProperty(HIVE_CATALOG, PARTIAL_AGGREGATION_PUSHDOWN_ENABLED, "false")
                 .build();
     }
+
+    @Test
+    public void testNegativeArraySubscriptPushdown()
```

Comment:
testArraySubscriptPushdown since you test both positive and negative subscripts here.

Optional, but I sometimes like to add a comment with the URL of the specific bug being addressed by the test. E.g.

// https://github.com/prestodb/presto/issues/22690

Reply:
Updated as per comment